### DISCUSSION TITLE
I'm trying to upgrade BB version but "Is In" filter doesn't work

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#14122] [Help] [answered]
I am currently using BB v2.21.9 on local deployment (Business plan), and trying to upgrade to the latest BB version. While testing my app on the BB cloud (v2.29.14) for that,  I have found:

- Consider this table with button names and corresponding permissions:
![image](https://github.com/Budibase/budibase/assets/93232582/4944d1e9-689a-4ff8-a88a-fb9893258c02)

- On my repeater block, I have a filter:
![image](https://github.com/Budibase/budibase/assets/93232582/2f3818a1-ef97-4a02-ab70-4260dc3a5bfa)

- On v2.21.9, I get this result:
![image](https://github.com/Budibase/budibase/assets/93232582/8ca3d1f9-06f2-4205-976f-9518c43b4b65)

- However, on v2.29.14, I get nothing:
![image](https://github.com/Budibase/budibase/assets/93232582/e68197ab-d848-419d-bcbf-2e3441f6ccbd)

- If I change the filter to use Javascript, it works again.
![image](https://github.com/Budibase/budibase/assets/93232582/d965b3ed-7ba1-48e2-bf28-772e7fe3faa4)

I think it is a bug in the new version. Because my app is using "Is In" filter in numerous locations, it is not practical to find them all and revise them to use Javascript. Is it a known problem? And I wonder when it will be fixed, because I can consider upgrade after this is solved.

Thanks in advance!
Danny Shin

**Comment by mikesealey** [comment, marked as answer]
Hey @dannyshin76 

We are aware of an issue with the "Is In" filter, and our engineering team are currently working on it. 

https://github.com/Budibase/budibase/issues/14127

**Reply by dannyshin76** [reply]
Got that, thanks!

**Comment by dannyshin76** [comment]
Just tried on v2.29.16 and saw this has been fixed. Thank you!
